display_list/display_list_ops.cc

```diff
@@ -0,0 +1,11 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/display_list/display_list_ops.h"
+
+namespace flutter {
+
+// Just exists to ensure that the header can be cleanly imported.
```

Comment:
b/212527291 <- looks like the internal compiler caught a problem w.r.t. this. Does display_list_ops.h need to include display_list_dispatcher.h?

Reply:
The internal compiler isn't supposed to compile that TU as display_list_ops.cc isn't in the GN definition. But, it should be as the point of that file is to make sure it imports cleanly (as the comment indicates). I guess the import process just copies the files and compiles them all. I'm going to mark that as wont-fix but also submit a patch to the GN rules that compiles this TU (and fix associated issues). That may fix that specific issue with the internal compiler. If there are other issues with the import process itself, I'll redirect them to Flutter@Google.